enterprise/api/v1/views/enterprise_group.py

```diff
@@ -157,18 +171,32 @@ def assign_learners(self, request, group_uuid):
             customer = group.enterprise_customer
         except models.EnterpriseGroup.DoesNotExist as exc:
             raise Http404 from exc
-        if requested_emails := request.POST.dict().get('learner_emails'):
+        if requested_emails := request.POST.getlist('learner_emails'):
+            request_data = {}
+            request_data['act_by_date'] = request.data.get('act_by_date')
+            request_data['catalog_uuid'] = request.data.get('catalog_uuid')
+            request_data['learner_emails'] = requested_emails
```

Comment:
You shouldn't need to do any of this - you should be able to pass `request.data` directly into the serializer like so:
```python
param_serializer = serializers.EnterpriseGroupAssignLearnersRequestDataSerializer(request.data)
param_serializer.is_valid(raise_exception=True) # will raise a validation error and cause a 400 response automatically
act_by_date = param_serializer.data['act_by_date']
# ...and so on
```
This is a pattern we've been using with success on the enterprise-access service, e.g. https://github.com/openedx/enterprise-access/blob/main/enterprise_access/apps/api/v1/views/subsidy_access_policy.py#L808